tests/testthat/test-visr_plot.R

```diff
@@ -199,11 +199,13 @@ testthat::test_that("T2.8 When `x_label` is `NULL` and the `survfit` object does
   
   names(survfit_object)[which(names(survfit_object) == "PARAM")] <- "visR"
   
-  gg <- survfit_object %>% visR::visr(y_label = NULL)
+  gg <- survfit_object %>% visR::visr(x_label = NULL)
   
   ggb <- ggplot2::ggplot_build(gg)
   
-  testthat::expect_true("time" %in% ggb$layout$panel_params[[1]]$x$name)
+  # Ensures that the label is pulled from gg$labels$x
+  testthat::expect_true("waiver" %in% class(ggb$layout$panel_params[[1]]$x$name))
+  testthat::expect_true("time" %in% gg$labels$x)
```

Comment:
Why do we need time in the label. The user is allowed to choose, we cant force that.

Reply:
![image](https://user-images.githubusercontent.com/12818242/120991346-d1d46b80-c781-11eb-884c-7cef09cfd0b4.png)

Because this "time" is the default given name and the test checks for that

Reply:
I dont agree with a default name. If the user does not specify anything and param in not in the dataset, then default should be NULL

Reply:
So leaving the axis unlabeled? I feel like this goes against our biased approach, no? Maybe we can talk about that during the next core group meeting :+1:

Reply:
I created an issue for it so we don't forget: https://github.com/openpharma/visR/issues/213

Reply:
Thanks. Leaving the axis unlabeled is better because you will see you missed something. We can trigger a warning.